pkgs/top-level/all-packages.nix

```diff
@@ -19471,6 +19471,8 @@ in
   };
 
   pulseaudio = callPackage ../servers/pulseaudio ({
+    inherit (darwin) Libc;
+    inherit (darwin.apple_sdk) sdk;
```

Comment:
`darwin.apple_sdk.sdk` doesn't exist on aarch64. See #131791

Reply:
Thanks for pointing this out. I think we're in luck. `libpulseaudio` doesn't actually seem to require the SDK, Libc and the frameworks suffice to build it.